app/javascript/admin/controllers/config_controller.js

```diff
@@ -93,6 +93,40 @@ export default class ConfigController extends Controller {
     }
   }
 
+  displaySnackbar(message) {
+    return document.dispatchEvent(
+      new CustomEvent('snackbar:add', { detail: { message } }),
+    );
+  }
+
+  async activateLightConfirmation(event) {
+    event.preventDefault();
+    try {
+      const data = new FormData(event.target);
+      const response = await fetch(event.target.action, {
+        method: 'POST',
+        headers: {
+          Accept: 'application/json',
+          'X-CSRF-Token': document.querySelector("meta[name='csrf-token']")
+            ?.content,
+        },
+        body: data,
+        credentials: 'same-origin',
+      });
+
+      const outcome = await response.json();
+
+      if (response.ok) {
+        this.displaySnackbar(outcome.message);
+      } else {
+        this.displaySnackbar(outcome.error);
+      }
+    } catch (err) {
+      this.displaySnackbar(err);
+    }
+    event.target.querySelector('input[type=submit]').disabled = false;
```

Comment:
For some reason, after `activateLightConfirmation` has fired, the `Update Settings` button remains disabled until after page-refresh. I added this line to re-enable it, so that if the user makes another change before refreshing the page, they can click the button again.

I welcome any solutions that are cleaner than this 🙏🏾